src/providers.ts

```diff
@@ -205,6 +208,11 @@ export function generateHoverString(flag: Flag, c: FlagConfiguration) {
 			hoverString = hoverString.appendCodeblock(`${field[1]}`);
 		}
 	});
+	if (url) {
+		hoverString.appendText('\n')
+		hoverString = hoverString.appendMarkdown(`[Open In Browser](${url})`)
```

Comment:
Let's use sentence case for this hyperlink
```suggestion
		hoverString = hoverString.appendMarkdown(`[Open in browser](${url})`)
```